erpnext/controllers/subcontracting.py

```diff
@@ -0,0 +1,342 @@
+from __future__ import unicode_literals
+
+import frappe
+from frappe import _
+from frappe.utils import flt, cint
+from collections import defaultdict
+from erpnext.stock.doctype.serial_no.serial_no import get_serial_nos
+
+class Subcontracting(object):
```

Comment:
```suggestion
class Subcontracting():
```
Passing `object` is [not needed in py3](https://stackoverflow.com/questions/7375595/class-with-object-as-a-parameter) as it was in py2